src/brevitas_examples/stable_diffusion/main.py

```diff
@@ -89,36 +111,102 @@ def main(args):
         if hasattr(m, 'lora_layer') and m.lora_layer is not None:
             raise RuntimeError("LoRA layers should be fused in before calling into quantization.")
 
+    # Move model to target device
+    print(f"Moving model to {args.device}...")
+    pipe = pipe.to(args.device)
+
+    if args.activation_equalization:
+        with activation_equalization_mode(pipe.unet, alpha=0.5, layerwise=True, add_mul_node=True):
+            # Workaround to expose `in_features` attribute from the Hook Wrapper
+            for m in pipe.unet.modules():
+                if isinstance(m, KwargsForwardHook) and hasattr(m.module, 'in_features'):
+                    m.in_features = m.module.in_features
```

Comment:
@costigt-dev This (and similarly, line 129-131) is why I need the wrapper to expose the attributes of the wrapped module.

In this case, the wrapping is only temporary so I need to keep some sort of reference of wrapper vs wrapped